### ISSUE TITLE
changed json?

### ISSUE THREAD

**Issue by vader2014** [s0faking/plugin.audio.radiothek#7]
Hi!

When I choose "Sendung Verpasst" and navigate to a podcast I get an error:

```
ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
 NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
Error Type: <type 'exceptions.TypeError'>
Error Contents: 'NoneType' object has no attribute '__getitem__' Traceback (most recent call last):
File "/home/pi/.kodi/addons/plugin.audio.radiothek/default.py", line 7, in <module> Addon.main(handle)
File "/home/pi/.kodi/addons/plugin.audio.radiothek/resources/lib/Addon.py", line 308, in main broadcasts = pi.get_broadcast_details(link)
File "/home/pi/.kodi/addons/plugin.audio.radiothek/resources/lib/RadioThek.py", line 460, in get_broadcast_details stream_url = self.get_stream_url(broadcast_details)
File "/home/pi/.kodi/addons/plugin.audio.radiothek/resources/lib/RadioThek.py", line 138, in get_stream_url(host, host_channel, loopStreamId, offset) = self.get_stream_base(station, json_item['start'], json_item['streams'])
File "/home/pi/.kodi/addons/plugin.audio.radiothek/resources/lib/RadioThek.py", line 127, in get_stream_base channel_infos = api_reference['stations'][station]
TypeError: 'NoneType' object has no attribute '__getitem__'
 -->End of Python script error report<--

NOTICE: PulseAudio: Opened device Default in pcm mode with Buffersize 450 ms
ERROR: GetDirectory - Error getting plugin://plugin.audio.radiothek/?link=https%3a%2f2faudioapi.orf.at%2foe1%2fapi%2fjson%2f4.0%2fbroadcast%2f691479%2f20220921&mode=broadcast_detail
ERROR: CGUIMediaWindow::GetDirectory(plugin://plugin.audio.radiothek/?link=https%3a%2f2faudioapi.orf.at%2foe1%2fapi%2fjson%2f4.0%2fbroadcast%2f691479%2f20220921&mode=broadcast_detail) failed

```

unfortunately i have no older json at hand but i would assume that the `host` information was removed?
if i manually open

`https://loopstream01.apa.at/?shoutcast=0&player=radiothek_v1&referer=radiothek.orf.at&offset=0&id=2022-09-21_0659_tl_51_7DaysWed6_1924633.mp3&channel=oe1`

it works.

**Comment by lomion0815** [comment]
They moved to https://sound.orf.at and changed the api there somehow. Is there any documentation about this api?

**Comment by s0faking** [comment]
should be fixed with 0.1.3

https://github.com/s0faking/plugin.audio.radiothek/releases/tag/0.1.3

**Comment by s0faking** [comment]
pushed 0.1.3 to the official repo
